src/components/about.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export const About = () => {
+  return (
+    <>
+      <h3>Secret Santa</h3>
+      <p>
+        The basic concept of the Secret Santa game is simple.
+        All of the participants' names are placed into a hat, box, etc. ...
+        When the gift-giving time arrives, each recipient finds his/her gift and must guess who their Secret Santa is.
+        If they can't guess, their Secret Santa eventually confesses.
+      </p>
+      <img src="https://i.pinimg.com/564x/cd/6e/01/cd6e01cc035f1422da18b8480a366cf2.jpg" className="img-fluid" alt="Santa"></img>
```

Comment:
`img` is considered a self closing tag, since it doesn't need to wrap text as many other tags do.

```suggestion
      <img src="https://i.pinimg.com/564x/cd/6e/01/cd6e01cc035f1422da18b8480a366cf2.jpg" className="img-fluid" alt="Santa" />
```